production.Dockerfile

```diff
@@ -106,16 +106,15 @@ RUN apk --update --no-cache --virtual .geolite-deps add \
 COPY requirements.txt ./
 RUN apk --update --no-cache --virtual .build-deps add \
     "bash~=5.1" \
-    "g++~=10.3" \
-    "gcc~=10.3" \
-    "cargo~=1.52" \
+    "g++~=11.2" \
+    "gcc~=11.2" \
+    "cargo~=1.60" \
     "git~=2" \
     "make~=4.3" \
-    "libffi-dev~=3.3" \
+    "libffi-dev~=3.4" \
     "libxml2-dev~=2.9" \
     "libxslt-dev~=1.1" \
     "xmlsec-dev~=1.2" \
-    "postgresql-dev~=13" \
```

Comment:
This package doesn't exist in Alpine 3.15 and 3.16. I'll get the error at build and see which other package we could use

Reply:
It has been renamed to `postgresql13-dev`